src/app.ts

```diff
@@ -598,9 +603,22 @@ app.post(
       return;
     }
 
-    const payload = JSON.parse(req.body.payload);
+    const payload: SlackInteractionEventPayload = JSON.parse(req.body.payload);
 
-    await enqueueBackgroundTask('slackInteractivityHandler', payload);
+    const metadata: InteractivityHandlerMetadata = {};
+
+    if (payload.type === 'message_action') {
```

Comment:
Can we add check the `callback_id` instead of just looking at `payload.type`? We may be adding additional message actions in the future that don't open a modal.